cinema/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
```

Comment:
The import statement for `SAFE_METHODS` is incorrect. It should be imported from `rest_framework.permissions` without the comma. Correct it to: `from rest_framework.permissions import BasePermission, SAFE_METHODS`.